### PR TITLE
Add hash type param to GenContainer tool

### PR DESCRIPTION
**Pull request by lsubash**
sigining hash type parameter added with command line
args for create, replace and sign component image.

Signed-off-by: Subash Lakkimsetti <subashx.lakkimsetti@intel.com>

### CODE REVIEW

File: BootloaderCorePkg/Tools/GenContainer.py [hunk 1]
```diff
@@ -734,6 +736,7 @@ def main():
 	cmd_display.add_argument('-c',  dest='compress', choices=['lz4', 'lzma', 'dummy'],  default='dummy', help='compression algorithm')
 	cmd_display.add_argument('-a',  dest='auth', choices=['rsa2048', 'sha256', 'none'], default='none',  help='authentication algorithm')
 	cmd_display.add_argument('-k',  dest='key_file',  type=str, default='', help='Private key file path to sign component')
+	cmd_display.add_argument('-ht', dest='hash_type', type=str, required=True, help='Hash Alg for signing')
```
Comment (mauricema) [inline BootloaderCorePkg/Tools/GenContainer.py]: -ht should be optional.   you should keep a default value if not specified.

Comment (lsubash) [reply BootloaderCorePkg/Tools/GenContainer.py]: Sure, i would update to "SHA2_256" as default one.

Comment (myguitar) [reply BootloaderCorePkg/Tools/GenContainer.py]: Can you give choices as well?

Comment (lsubash) [reply BootloaderCorePkg/Tools/GenContainer.py]: Currently we have only SHA256 support. While adding SHA384 support in signing, I could add it.

Comment (mauricema) [reply BootloaderCorePkg/Tools/GenContainer.py]: I am fine with it.

**Pull request merged**